src/components/OSCALControl.js

```diff
@@ -61,6 +61,21 @@ export default function OSCALControl(props) {
     );
   }
 
+  // When FedRAMP, retrieve the set-parameters from the initial implementation request statement
+  let firstImplReqStatement = null;
```

Comment:
Why are we just pulling the first of each here?

Reply:
Yes. In this sample it contains the `set-parameters` only in the first (initial) implemented requirement statement

Reply:
Right, but we don't have guaranteed ordering here.

What I think we're really looking for are `set-parameters` at the 'top-level' of the control, so within `ac-1_smt`, which won't necessarily be the first statement in the array.

Reply:
Oh okay, that makes more sense. Maybe instead here, we should find the 'top-level' of the control with an ends with "_smt" check?

Reply:
Something like that, we may already be doing something similar elsewhere?

Reply:
I've added an endsWith check when searching through the array of `implReqStatements` for now. I haven't found a place where we're doing something similar but will continue looking.